src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Feedback/DefaultFeedbackFileLogWriter.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Globalization;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.Feedback
+{
+    internal class DefaultFeedbackFileLogWriter : FeedbackFileLogWriter, IDisposable
```

Comment:
This class is highly influenced from [LiveShare's file log writer](https://devdiv.visualstudio.com/DevDiv/_git/Cascade?path=%2Fsrc%2FTracing%2FTracing%2FLogFileTraceListener.cs).